src/Stratis.Bitcoin.Features.Consensus.Tests/PowConsensusRulesRegistrationTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using FluentAssertions;
+using Stratis.Bitcoin.Features.Consensus.Rules;
+using Stratis.Bitcoin.Features.Consensus.Rules.CommonRules;
+using Xunit;
+
+namespace Stratis.Bitcoin.Features.Consensus.Tests
+{
+    public class PowConsensusRulesRegistrationTests
+    {
+        private readonly IEnumerable<ConsensusRule> rules;
+
+        public PowConsensusRulesRegistrationTests()
+        {
+            this.rules = new FullNodeBuilderConsensusExtension.PowConsensusRulesRegistration().GetRules();
+        }
+
```

Comment:
Could get rid of the constructor and class level variable and move into the test it self.